dkron/config.go

```diff
@@ -90,39 +90,39 @@ func ConfigFlagSet() *flag.FlagSet {
 	c := DefaultConfig()
 	cmdFlags := flag.NewFlagSet("agent flagset", flag.ContinueOnError)
 
-	cmdFlags.Bool("server", false, "start dkron server")
-	cmdFlags.String("node-name", c.NodeName, "node name")
-	cmdFlags.String("bind-addr", c.BindAddr, "address to bind listeners to")
-	cmdFlags.String("advertise-addr", "", "address to advertise to other nodes")
-	cmdFlags.String("http-addr", c.HTTPAddr, "HTTP address")
-	cmdFlags.String("discover", c.Discover, "mDNS discovery name")
+	cmdFlags.Bool("server", false, "This node is running in server mode.")
+	cmdFlags.String("node-name", c.NodeName, "Name of this node. Must be unique in the cluster.")
+	cmdFlags.String("bind-addr", c.BindAddr, "Address to bind network listeners to.")
+	cmdFlags.String("advertise-addr", "", "Address used to advertise to other nodes in the cluster. By default, the bind address is advertised.")
+	cmdFlags.String("http-addr", c.HTTPAddr, "Address to bind the UI web server to. Only used when server.")
+	cmdFlags.String("discover", c.Discover, "A cluster name used to discovery peers. On networks that support multicast, this can be used to have peers join each other without an explicit join.")
```

Comment:
I think you can remove or comment it as the feature is no developed, it's missleading